roaring/roaring.go

```diff
@@ -1868,16 +1868,26 @@ func intersectionCount(a, b *Container) int32 {
 }
 
 func intersectionCountArrayArray(a, b *Container) (n int32) {
-	na, nb := len(a.array), len(b.array)
-	for i, j := 0, 0; i < na && j < nb; {
-		va, vb := a.array[i], b.array[j]
-		if va < vb {
-			i++
-		} else if va > vb {
-			j++
-		} else {
+	s1, s2 := a.array, b.array
```

Comment:
There's a de facto convention for roaring variable names, and there's a new convention being introduced here, because of the swap. Whatever that is, can we settle on something and stick with it, in any future swap logic?

I might have used `n1` instead of `l1`, and maybe `c1, c2` - `c` for container.

Reply:
good catch, i'll fix that.

Reply:
on consideration, it's better to just use the existing convention, and just let the swap be a swap of a and b. we already have these implicitly in a number of places; an intersection count between "a" and "b" can be implemented by passing them to a concrete intersection count function in either order, so it's not breaking anything to just say "without loss of generality, let a be the shorter array".